manuscript/main.tex

```diff
@@ -527,19 +527,33 @@ \subsubsection*{Gut microbial taxonomic and functional profiles predict cognitiv
     \centering
     \includegraphics[width=\textwidth]{assets/Figure3.png}
     \caption{
-        Random forest models predict concurrently measured cognitive function.
+        Random forest models predict both concurrent and future measured cognitive function.
         Comparison of RF predictor importance versus linear models for children
-        between birth and 6 months old (\textbf{A}), and for those older than 18 months
-        (\textbf{B}). Colors represent whether the species belong to the group of
+        over 18 months (\textbf{A}).
+        Colors represent whether the species belong to the group of
         top-important features that account for 60\% of the cumulative
         importance on the RF model, if that species was significant (q
-        \textless{} 0.2) in linear models, both, or neither. Ranked feature
-        importance for taxa in RF models for children between birth and 6 months
-        old (\textbf{C}), and for those older than 18 months (\textbf{D}). Taxa that are important
-        for RF models both for children under 6 months and for those over 18
-        months (\textbf{E}), only for children under 6 months (\textbf{F}), and only for children
-        over 18 months (\textbf{G})
-    }
+        \textless{} 0.2) in linear models, both, or neither.
+        Comparison of RF predictor importance for microbial features
+        between concurrent and future cognitive function measure
+        (\textbf{B}).
+        Colors represent whether the species belong to the group of
+        top-important features that account for 60\% of the cumulative
+        importance on the RF model only for concurrent prediction,
+        only for the future prediction, or for both models.
```

Comment:
This repeats 532-536. I think it's obvious, but you could also say "colored as in A" or something

Reply:
I do not think this is correct. A says
"Colors represent whether the species belong to the group of
        top-important features that account for 60\% of the cumulative
        importance on the RF model, **if that species was significant (q
        \textless{} 0.2) in linear models, both, or neither.**"

and B says
"Colors represent whether the species belong to the group of
        top-important features that account for 60\% of the cumulative
        importance on the RF model **only for concurrent prediction,
        only for the future prediction, or for both models.**"

so there is a diference. A compares two different measures for predicting the same thing, B compares the same measure for predicting two different things

Reply:
Oh I see - glossed over that. Fine by me